addon/components/frost-select.js

```diff
@@ -206,14 +221,12 @@ export default Ember.Component.extend({
   /* Ember.didInsertElement method */
   didInsertElement () {
     this._super(...arguments)
-    const outsideClickHandler = this.get('handleOutsideClick').bind(this)
-    Ember.$(document).on('click', outsideClickHandler)
+    Ember.$(document).on('click', handleOutsideClick.bind(this))
   },
 
   /* Ember.willDestroyElement method */
   willDestroyElement () {
-    const outsideClickHandler = this.get('handleOutsideClick').bind(this)
-    Ember.$(document).on('click', outsideClickHandler)
+    Ember.$(document).off('click', handleOutsideClick.bind(this))
```

Comment:
@nskrypnik This will never work because `bind` returns a new function.  You'd need to pass the same handler that was passed to `on`.